CMSgov/design-system
# [WNMGDS-480] Add usa banner component

### Added
- Adding new USA Banner component
- Adding SVG icons for USA Flag, HTTPs Icon, Lock Icon and Dot Gov Icon, which are also used in the banner
- Added documentation page for USA Banner as well as guidance 
- Added HTML and React examples 

### How To test
- Run the doc site locally with `yarn start`
- Then visit the USA Banner component page and ensure that the content expands and collapses properly and the styling is correct.
